packages/ckeditor5-mention/tests/mentionui.js

```diff
@@ -299,6 +299,40 @@ describe( 'MentionUI', () => {
 					expect( limiter() ).to.be.null;
 				} );
 		} );
+
+		describe( 'contextual balloon in rtl', () => {
+			let contextualBaloonSpy;
+
+			beforeEach( async () => {
+				await editor.destroy();
+
+				return createClassicTestEditor( { ...staticConfig } )
+					.then( () => {
+						const contextualBalloon = editor.plugins.get( ContextualBalloon );
+						setData( model, '<paragraph>foo []</paragraph>' );
+						editor.locale.uiLanguageDirection = 'rtl';
+						contextualBaloonSpy = sinon.spy( contextualBalloon, 'add' );
+
+						model.change( writer => {
+							writer.insertText( '@', doc.selection.getFirstPosition() );
+						} );
+					} )
+					.then( waitForDebounce );
+			} );
+
+			it( 'should set proper position list in rtl', () => {
```

Comment:
We're missing another test next to this one with LTR for symmetry, as per my comment https://github.com/ckeditor/ckeditor5/pull/13443#discussion_r1105700819